plugins/blockchain_statistics/plugin.cpp

```diff
@@ -568,44 +342,239 @@ void plugin::plugin_startup() {
+    result.push_back("new_reply_votes:" + std::to_string(new_reply_votes));
+    result.push_back("changed_reply_votes:" + std::to_string(changed_reply_votes));
+    result.push_back("payouts:" + std::to_string(payouts));
+    result.push_back("sbd_paid_to_authors:" + std::string(sbd_paid_to_authors));
+    result.push_back("vests_paid_to_authors:" + std::string(vests_paid_to_authors));
+    result.push_back("vests_paid_to_curators:" + std::string(vests_paid_to_curators));
+    result.push_back("liquidity_rewards_paid:" + std::string(liquidity_rewards_paid));
+    result.push_back("transfers_to_vesting:" + std::to_string(transfers_to_vesting));
+    result.push_back("steem_vested:" + std::string(steem_vested));
+    result.push_back("new_vesting_withdrawal_requests:" + std::to_string(new_vesting_withdrawal_requests));
+    result.push_back("modified_vesting_withdrawal_requests:" + std::to_string(modified_vesting_withdrawal_requests));
+    result.push_back("vesting_withdraw_rate_delta:" + std::string(vesting_withdraw_rate_delta));
+    result.push_back("vesting_withdrawals_processed:" + std::to_string(vesting_withdrawals_processed));
+    result.push_back("finished_vesting_withdrawals:" + std::to_string(finished_vesting_withdrawals));
+    result.push_back("vests_withdrawn:" + std::string(vests_withdrawn));
+    result.push_back("vests_transferred:" + std::string(vests_transferred));
+    result.push_back("sbd_conversion_requests_created:" + std::to_string(sbd_conversion_requests_created));
+    result.push_back("sbd_to_be_converted:" + std::string(sbd_to_be_converted));
+    result.push_back("sbd_conversion_requests_filled:" + std::to_string(sbd_conversion_requests_filled));
+    result.push_back("steem_converted:" + std::string(steem_converted));
+    result.push_back("limit_orders_created:" + std::to_string(limit_orders_created));
+    result.push_back("limit_orders_filled:" + std::to_string(limit_orders_filled));
+    result.push_back("limit_orders_cancelled:" + std::to_string(limit_orders_cancelled));
+    result.push_back("total_pow:" + std::to_string(total_pow));
+    result.push_back("num_pow_witnesses:" + std::to_string(num_pow_witnesses));
+    return result;
 }
 
-    void plugin::plugin_shutdown() {
+std::vector < std::string > runtime_bucket_object::calculate_delta_with (const runtime_bucket_object & b) {
+    std::vector < std::string > result;
+    if (b.seconds - seconds != 0 ) {
+        result.push_back("seconds:" + std::to_string(b.seconds - seconds) + "|c");
     }
-
-
-    std::string increment_counter(std::string name) {
-    std::string res = name + ":1|c";
-    return res;
-}
-
-std::string increment_counter(std::string name, uint32_t value) {
-    std::string res = name + ":";
-    std::string num = std::to_string(value);
-    res += num + "|c";
-
-    return res;
+    if (b.blocks - blocks != 0 ) {
+        result.push_back("blocks:" + std::to_string(b.blocks - blocks) + "|c");
+    }
```

Comment:
Return the increment_counter() - it's more readable, than now:
```
...
increment_counter(result, "blocks", b.blocks - blocks);
increment_counter(result, "bandwidth", b.bandwidth - bandwidth);
...
```
Where increment_function() is:
```
void increment_counter(std::vector < std::string > &result, std::string name, uint32_t value) {
    if (value != 0) {
         result.push_back(name + std::to_string(value) + "|c");
    }
}
```

Reply:
Done